src/_locales/de/messages.json

```diff
@@ -545,43 +545,43 @@
         "description": "Label for a dropdown control on the Tracking Domains options page tab."
     },
     "options_domain_filter_all": {
-        "message": "alle",
+        "message": "Alle",
```

Comment:
This and the following lowercase messages are options in dropdown filters on the options page. They are lowercase in the source text on purpose.

Reply:
Done